<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manualLEcontrol</class>
 <widget class="QWidget" name="manualLEcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Manual Light Engine Control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="InternalMemoryTest">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Internal Memory Test Passed</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ControllerDMD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller/DMD Compatible</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="SlavePresent">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Slave Present and Ready</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="InternalInit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Internal Initialization Complete</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="DMDReset">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DMD Reset Waveform Controller</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="SequencerAbortFlag">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequencer Abort Status Flag</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ForcedSwapErrorTest">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Forced Swap Error Test Passed</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="DMDParked">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DMD Micromirrors Parked</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="SequencerErrorTest">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequencer Error Test Passed</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="LockedToExternal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Locked to External Source</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="VideoRunning">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Running</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="SequencerRunning">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sequencer Running</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
